tests/playwright-test/fixtures.spec.ts

```diff
@@ -714,30 +714,3 @@ test('worker teardown errors reflected in timed-out tests', async ({ runInlineTe
   expect(result.output).toContain('Test timeout of 1000ms exceeded.');
   expect(result.output).toContain('Rejecting!');
 });
-
-test('should not complain about fixtures of non-focused tests', async ({ runInlineTest }) => {
```

Comment:
We can keep this functionality by splitting `buildPools` and `validateSignatures` steps and filtering in-between.

Reply:
This won't work when loader is out of process sadly.